### DISCUSSION TITLE
Loading Issue

### DISCUSSION THREAD

**Discussion by Samubrine** [ORelio/Minecraft-Console-Client#1828] [Q&A]
I used console client back in 2020 and just came back recently with a new PC, and I decided to open it and I realized that it takes roughly ~20 minutes for it to actually join the server.  I have no idea what's causing consoleclient to be very slow and I have not found any solution for this problem yet. Please help

**Comment by ReinforceZwei** [comment]
Please give some screenshots or other materials to illustrate your problem.

**Reply by Samubrine** [reply]
![image](https://user-images.githubusercontent.com/52322527/142901095-c8537ce3-8521-4cba-890a-ff45e89ee012.png)
The client just gets stuck at this and will finish loading after 5-30minute

**Reply by ReinforceZwei** [reply]
Could you please enable debug messages in the config and post another screenshot again?

**Reply by Samubrine** [reply]
![image](https://user-images.githubusercontent.com/52322527/142959253-c7641575-6b2e-415d-9af7-4372e3af53ca.png)

**Reply by ORelio** [reply]
Seems like MCC is stuck with Minecraft profiles. Try going into `%appdata%\.minecraft` (https://www.wikihow.com/Access-Your-.Minecraft-Folder-on-the-Computer) and move away `launcher_profiles.json`, then launch MCC again. If it works, then we can have a look there. Anyway, you can put back the `launcher_profiles.json` into `.minecraft` once done.
